src/pages/index.js

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { Link } from "gatsby"
+import styles from "./styles/index.module.css"
+import { graphql } from "gatsby"
+
+import Layout from "../components/layout"
+
+export default function Home(props) {
+
+  console.log(props.data.allFile)
+
+  return (
+  <Layout>
+    <div
+    className={styles.headerBox}
+    >
+      <h1
+      className={styles.header}
+      >
+          {props.data.site.siteMetadata.title}
+      </h1>
+    </div>
+  </Layout>
+  )
+}
```

Comment:
Might be a good idea to try out `PropTypes` on at least a few of these components. It helps you enforce that components are being given the props you expect, and throws console errors when a required prop is missing or the wrong type.

It's not as good as the type-checking something like TypeScript does, but it's a really quick addition to React code in standard JS that can help you catch issues.

[More info](https://reactjs.org/docs/typechecking-with-proptypes.html)